src/runner.ts

```diff
@@ -29,7 +29,7 @@ const processOptions = async (options: RunnerOptions) => {
     }
 
     assert(
-      !!options.pgPool !== !!options.connectionString,
+      !(!!options.pgPool && !!options.connectionString),
```

Comment:
To avoid the double-negatives, could we go with something like:

```suggestion
      !options.pgPool || !options.connectionString,
```

Other than that, this looks good to merge :+1: